src/main/scala/nl/knaw/dans/easy/stage/Conf.scala

```diff
@@ -70,7 +70,6 @@ class Conf(args: Seq[String]) extends ScallopConf(args) {
     name = "staged-digital-object-set",
     descr = "The resulting Staged Digital Object directory (will be created if it does not exist)",
     required = true)(mayNotExist)
-  codependent(urn,doi)
```

Comment:
Well, that is what `codependent` means, right? At least according to the [ScalaDoc about Scallop](http://scallop.github.io/scallop/#org.rogach.scallop.ScallopConf).